plugins/nitrokey/fu-nitrokey-device.c

```diff
@@ -13,26 +13,31 @@
 
 G_DEFINE_TYPE (FuNitrokeyDevice, fu_nitrokey_device, FU_TYPE_USB_DEVICE)
 
+typedef struct {
+	guint8		 command;
+	const guint8	*buf_in;
+	gsize		 buf_in_sz;
+	guint8		*buf_out;
+	gsize		 buf_out_sz;
+	GCancellable	*cancellable;
```

Comment:
I'm not sure plumbing cancellable all the way through makes sense.  It's `NULL` at it's earliest pass in.  Maybe just drop it as part of this?

Reply:
Fixed, thanks. I also noticed it was another SET_REPORT/GET_REPORT conversion, so I'll do that as a follow up.